### PR TITLE
Epic/multiple files to a dataset

### PR DESCRIPTION
**Pull request by mickel1138**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -10,6 +10,7 @@ These instructions will get you a copy of the project up and running on your loc
 ### Prerequisites
 
 * [CENTOS 7](https://wiki.centos.org/) - Linux Distribution based on Red Hat
+* [ClamAV](https://www.clamav.net) - ClamAV® open source antivirus engine
 * [PHP 7.1+](http://php.net/docs.php) - General-purpose scripting language
 * [PostgreSQL 9.6+](https://www.postgresql.org/docs/9.6/static/release-9-6.html) - ORDBMS
 * [Elasticsearch 6.7.2+](https://www.elastic.co/products/elasticsearch) - ElasticSearch Document Indexer
```
Comment (mickel1138) [inline README.md:16]: Rabbit can go.

**Review by mickel1138 — COMMENTED** [review]
Reviewed all files. LGTM

**Review by praneethpr — APPROVED** [review]
Tested, ready to merge!

**Review by fightingtexasaggie — APPROVED** [review]
reviewed in group

**Review by praneethpr — APPROVED** [review]

**Pull request merged**